src/Symfony/Component/Notifier/Bridge/Lox24/composer.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "symfony/lox24-notifier",
+  "type": "symfony-notifier-bridge",
+  "description": "Symfony LOX24 Notifier Bridge",
+  "keywords": ["sms", "lox24", "notifier"],
+  "homepage": "https://symfony.com",
+  "license": "MIT",
+  "require": {
+    "php": ">=8.1",
+    "symfony/http-client": "^6.4|^7.0",
+    "symfony/notifier": "^7.1"
+  },
+  "require-dev": {
+    "symfony/webhook": "^6.4|^7.0"
+  },
+  "autoload": {
+    "psr-4": {
+      "Symfony\\Component\\Notifier\\Bridge\\Lox24\\": ""
+    },
+    "exclude-from-classmap": [
+      "/Tests/"
+    ]
+  },
+  "scripts": {
+    "php-cs-fixer": "php-cs-fixer --config=./.php_cs"
+  },
+  "minimum-stability": "dev"
+}
```

Comment:
missing new line and the indention is 2, but we prefer 4 spaces. You can check other bridges for reference

Reply:
fixed